auth-web/tests/unit/components/SubProductSelector.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { Wrapper, mount } from '@vue/test-utils'
+import { SubProductConfigIF } from '@/models/SubProductConfigIF'
+import SubProductSelector from '@/components/auth/common/SubProductSelector.vue'
+import Vue from 'vue'
+import VueCompositionAPI from '@vue/composition-api'
+import Vuetify from 'vuetify'
+
+// @ts-ignore
```

Comment:
Why do you need this? I think we should not be disabling tools in the unit tests.